homeassistant/components/enphase_envoy/switch.py

```diff
@@ -95,6 +121,20 @@ async def async_setup_entry(
             for relay in envoy_data.dry_contact_status
         )
 
+    if (
+        envoy_data.enpower
+        and envoy_data.tariff
+        and envoy_data.tariff.storage_settings
+        and (coordinator.envoy.supported_features & SupportedFeatures.ENCHARGE)
+    ):
+        entities.extend(
+            [
+                EnvoyStorageSettingsSwitchEntity(
+                    coordinator, CHARGE_FROM_GRID_SWITCH, envoy_data.enpower
+                )
+            ]
+        )
```

Comment:
isn't `entities.extend([...])` the same as `entities.append(...)`

Reply:
append would be cleaner here since its just one